content/blog/deploying-super-mario-game-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,477 @@
+  tetris.example.com
+                           /   tetris-service:80 (10.192.3.231:3000,10.192.4.27:3000)
+Annotations:               cert-manager.io/issuer: cfe-selfsinged-issuer
+                           ingress.kubernetes.io/ssl-redirect: true
+Events:
+  Type    Reason             Age   From                       Message
+  ----    ------             ----  ----                       -------
+  Normal  Sync               30s   nginx-ingress-controller   Scheduled for sync
+  Normal  CreateCertificate  30s   cert-manager-ingress-shim  Successfully created Certificate "cfe-tls-key-pair"
+```
+
+### Access deployed games
+
+B﻿efore accessing the deployed games, you need set up the subdomain name resolution. For the subdomains, *super-mario.example.com* and *tetris.example.com*, the workstation host file has been used for DNS resolution.   
+
+Type the following commands to check that the domain/subdomain name resolution is set up correctly:  
+
+```shell
+$ host super-mario.example.com
+super-mario.example.com has address 10.6.115.251
+
+$ host tetris.example.com
+tetris.example.com has address 10.6.115.251
+```
+
+You can then access the deployed games using the browser. S﻿tart the browser and type the URL *super-mario.example.com*, it will be redirected over HTTPS with the warning message *'Your connection is not private'*: 
+
+![](/img/mario-private.png)
+
+T﻿his is due to the fact that the self-signed certifcate is generated in cert-manager and configured in the K8s Ingress resource.
+
+C﻿lick *Not secure* and start the Certificate Viewer to check the certificate:
+
+![](/img/mario-certificate.png)
+
+C﻿lick *Proceed to super-mario.example.com (unsafe)*, you then land to the *SUPER MARIO* game page: 
+
+![](/img/super-mario.png)
+
+If you type the URL *tetris.example.com* to the browser, it will be redirected over HTTPS with the same warning message *'Your connection is not private'*:  
+
+![](/img/tetris-private.png)
+
+C﻿lick *Proceed to tetris.example.com (unsafe)*, you then go to the Tetris *Start* page:
+
+![](/img/tetris-start.png)
+
+C﻿lick *Start* button, you then land to the *Tetris*  game page:
+
+![](/img/tetris.png)
+
+E﻿njoy playing your games !
+
+### Conclusion
+
+This blog post provided a comprehensive guide on how to d﻿eploy *Super Mario* and *Tetris* in a K8 cluster and e﻿xpose those games to be securely accessed via HTTPS in HPE GreenLake for Private Cloud Enterprise. It detailed the process of configuring TLS termination on an Ingress controller at the load balancer setup, utilizing a K8s Ingress resource and a self-signed TLS certificate generated with cert-manager. This guide fully aligns with the stringent security and compliance requirements of the K8s production environment in HPE GreenLake for Private Cloud Enterprise. 
+
+Please keep coming back to the [HPE Developer Community blog](https://developer.hpe.com/blog/) to learn more about HPE GreenLake for Private Cloud Enterprise.
```

Comment:
I'm tempted to have you add something like:
Please keep coming back to the HPE Developer Community blog to learn more about HPE GreenLake for Private Cloud Enterprise and get more ideas on how you can use it in your everyday operations."